modules/ROOT/pages/runtime-fabric-setup.adoc

```diff
@@ -19,7 +19,7 @@ Anypoint Visualizer relies on a header injection to display connections between
 
 == Enable Header Injection
 
-Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact in Mule 4, header injection is disabled by default.
+Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact, header injection is disabled by default in Mule 4.x released before April 7, 2020.
```

Comment:
At least this one clarifies a little bit: RELEASED before
I still wonder why this is not a "version"....

Reply:
```suggestion
Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact, header injection is disabled by default in Mule 4.x version released before April 7, 2020.
```